specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/examples/ClusterExtensionTypes_List.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "subscriptionId": "aaaaaaaaaa",
+    "resourceGroupName": "rgextensionTypes",
+    "clusterRp": "Microsoft.ContainerService",
+    "clusterName": "aaaaaaaaaaaaaaaaaaaaa",
+    "api-version": "2021-05-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "releaseTrains": [
+                "aaaaaaaaaaaaaaaaaaaaaa"
+              ],
+              "clusterTypes": "connectedClusters",
+              "supportedScopes": {
+                "defaultScope": "aaaaaaaaaaa",
+                "clusterScopeSettings": {
+                  "properties": {
+                    "allowMultipleInstances": true,
+                    "defaultReleaseNamespace": "aaaaaaaaaaaaaaaaaaaaaaa"
+                  },
+                  "id": "aaaaaaaaaaaaaaaa",
+                  "name": "aaaaaaaaaa",
+                  "type": "aaaaaaaaaaaaaaaaaaa"
+                }
+              }
+            },
+            "systemData": {
+              "createdBy": "aaaaaaaa",
+              "createdByType": "User",
+              "createdAt": "2021-04-14T22:44:27.144Z",
+              "lastModifiedBy": "aaaaaaaaa",
```

Comment:
>lastModifiedBy [](http://example.com/codeflow?start=15&length=14)

Since there is no PUT will there be ever modifiedBy?

Reply:
We will have PUT in a future release